lib/tests/2wp.js

```diff
@@ -139,6 +207,34 @@ const assert2wpBalancesAfterSuccessfulPegin = async (initial2wpBalances, peginVa
 
 };
 
+const addInputs = (tx, senderUtxosInfo) => {
+  senderUtxosInfo.utxos.forEach(utxo => {
+    tx.addInput(Buffer.from(utxo.txid, 'hex').reverse(), utxo.vout);
+  });
+};
+
+const addChangeOutputs = (tx, changeAddress, changeInSatoshis) => {
+  if(changeInSatoshis > 0) {
+    tx.addOutput(
+      bitcoinJsLib.address.toOutputScript(changeAddress, btcTxHelper.btcConfig.network),
+      changeInSatoshis - btcFeeInSatoshis
+    );
+  }
+};
+
+const ensurePeginIsPushed = async (btcPeginTxHash, expectedUtxosCount = 1) => {
```

Comment:
I would like this function to return the peginRskTxHash, wdyt?

Reply:
Out of the scope. It would complicate things quite a bit.